.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -4,6 +4,8 @@ about: Found a bug in OpenTTD?
 title: "Bug Report"
 ---
 
+<!-- Please fill in the sections below, without deleting any text. -->
```

Comment:
people do not so much delete text. I think the point was that people fill it in after `## Version of OpenTTD`, instead of on a new line. I think that is more important to hint, where to fill in the text. No suggestions how to word that :P

Reply:
My logic was telling them not to delete text would spur them to use the headers we provide, instead of clearing the form and typing wherever. Or do they leave the headers and just type above them?

(Also, I swear most of my squashed commits are "fix whitespace again"...sorry for spamming IRC with update notifications :) )

Reply:
Worse: after them. So: `## Version of OpenTTD - 1.11.1` shit.

This is also what nielsm suggests, to change `Indicate` into `fill in below` or something.

Reply:
Aha, I can do that and fix #9103 in one fell swoop.